From 28ab9480b0b40d35c64c0137777320b21739425d Mon Sep 17 00:00:00 2001
From: Guo Yi <yi.guo@cavium.com>
Date: Fri, 5 Jun 2020 12:16:18 -0700
Subject: [PATCH 1167/1239] octeontx2_95xx_defconfig: Add Winbond SPI flash
 support

CNF95xx-CRB is equipped with winbond SPI flash

Signed-off-by: Guo Yi <yi.guo@cavium.com>
Change-Id: Ia1954fcd0e4ee0a7ca0f517fbe7e347278fcb3cc
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/boot/u-boot/+/29499
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Suneel Garapati <sgarapati@caviumnetworks.com>
---
 configs/octeontx2_95xx_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/octeontx2_95xx_defconfig b/configs/octeontx2_95xx_defconfig
index 555817b5b7..c3c5056fa4 100644
--- a/configs/octeontx2_95xx_defconfig
+++ b/configs/octeontx2_95xx_defconfig
@@ -84,6 +84,7 @@ CONFIG_SPI_FLASH_SFDP_SUPPORT=y
 CONFIG_SPI_FLASH_MACRONIX=y
 CONFIG_SPI_FLASH_SPANSION=y
 CONFIG_SPI_FLASH_STMICRO=y
+CONFIG_SPI_FLASH_WINBOND=y
 CONFIG_DM_ETH=y
 CONFIG_NET_OCTEONTX2=y
 CONFIG_PCI=y
-- 
2.29.0

